src/data/markdown/docs/30 xk6-browser/30 xk6-browser.md

```diff
@@ -45,7 +45,7 @@ The first step is to launch a [Browser](/javascript-api/xk6-browser/browser). Af
 
 | Method                          | Description                                                                                                                                                                                 |
 |---------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| launch(browser_type, [options]) | The first argument is required and the only `browser_type` that is currently accepted is `chromium`. Below you will find a list of all possible `options`. The `options` param is optional. |
+| launch(browserType, [options]) | The first argument is required and the only `browserType` that is currently accepted is `chromium`. Below you will find a list of all possible `options`. The `options` param is optional. |
```

Comment:
```suggestion
| launch(browserName, [options]) | The first parameter is required, and the only accepted `browserName` is currently `chromium`. The `options` parameter is optional. You can see a list of all possible `options` in the options heading.
```

By saying the "options heading" instead of "below", we'd be invincible even if the layout changes.

Reply:
Yep, all sounds good 👍